api/models/Notification.js

```diff
@@ -1,31 +1,15 @@
 module.exports = bookshelf.Model.extend({
-  tableName: 'notification',
 
-  actor: function() {
-    return this.belongsTo(User);
-  },
+  tableName: 'notifications',
```

Comment:
@levity - someone has been here before me! Looks like this is old work so I'm clobbering it.